ui/src/app/views/sliding-fab/component.css

```diff
@@ -0,0 +1,34 @@
+.sliding-button {
+	box-shadow: 0 2px 5px 0 rgba(0,0,0,0.26), 0 2px 10px 0 rgba(0,0,0,0.16);
```

Comment:
Nit: Spacing

Reply:
Also, where is the box shadow coming from? If its coming from Invision, I would check with Lou, because I've noticed random box shadows sometimes showing up, where, when I talk to Lou, they shouldn't actually be there.

Reply:
Done - and double-checking with @lstrano-broad on the box-shadow. I would definitely expect a shadow on a floating button personally but I wasn't aware of a recurring problem of unexpected box shadows in invision.

Reply:
Verified with Lou that the shadow is intentional.